test/box/gh-6634-different-log-on-tuple-new-and-free.test.lua

```diff
@@ -0,0 +1,31 @@
+env = require('test_run')
+test_run = env.new()
+
+--
+s = box.schema.create_space('test')
+_ = s:create_index('pk')
+
+default_log_lvl = box.cfg.log_level
+box.cfg{log_level = 7}
+
+_ = s:replace{0, 0, 0, 0, 0, 0, 0}
+str = test_run:grep_log('default', 'tuple_new%(%d+%) = 0x%x+$')
```

Comment:
Actually, you don't need to grep logs at all. Firstly, imho it's not best practice; secondly - much more easier is to rely on box.slab.stats().

Reply:
But this test is about log...